sources/packages/web/src/views/aest/Login.vue

```diff
@@ -71,6 +72,7 @@ export default defineComponent({
     },
   },
   setup() {
+    sessionStorage.setItem(USER_LOGIN_TRIGGERED, "true");
```

Comment:
Should we add it on the click of the login button? Same in other `Login.vue` files.